fetch/main.js

```diff
@@ -0,0 +1,11 @@
+async function ThrowPokeball() {
+  const response = await fetch('https://pokeapi.co/api/v2/pokemon/151');
```

Comment:
no need to call fetch inside a function.

Reply:
Please explain at the next class!